src/uwtools/tests/test_rocoto.py

```diff
@@ -129,6 +129,70 @@ def test__add_task_dependency_fail(self, instance, root):
         with raises(UWConfigError):
             instance._add_task_dependency(e=root, config=config)
 
+    def test__add_task_dependency_and(self, instance, root):
+        config = {"and": {"or": {"datadep": {"attrs": {"age": "120"}}}}}
+        instance._add_task_dependency(e=root, config=config)
+        dependency = root[0]
+        assert dependency.tag == "dependency"
+        and_ = dependency[0]
+        assert and_.tag == "and"
+        # assert and_.get("age") == "120"
+
+    def test__add_task_dependency_streq(self, instance, root):
+        config = {"streq": {"attrs": {"left": "&RUN_GSI;", "right": "YES"}}}
+        instance._add_task_dependency(e=root, config=config)
+        dependency = root[0]
+        assert dependency.tag == "dependency"
+        streq = dependency[0]
+        assert streq.tag == "streq"
+        assert streq.get("left") == "&RUN_GSI;"
+
+    # pylint: disable=line-too-long
```

Comment:
I removed this line and the other on line 163 and all tests pass, so I think these are not needed.

Also, there's special `pylint` syntax (`pytlint: disable-next=`) for suppressing a complaint about just _the following_ line, and that's what we'd probably want to put on a line that we just couldn't wrap. A directive like this will suppress the warning for everything in the scope, which in this case would, I think, mean the entire `Test_RocotoXML` class.

Reply:
I removed those as well. I originally used them to work on my test cases and now that they'll _almost_ finalized we can remove them. I'll push those changes now and make this a full PR.